### DISCUSSION TITLE
Welcome to Real-Time-Human-Detection-Counting Discussions!

### DISCUSSION THREAD

**Discussion by akash-rajak** [akash435/Real-Time-Human-Detection-and-Counting#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sharkyangliu11** [comment]
It is really good software. What main techniques have been used in the software? Yolo version 5?.... Thank you very much.

**Comment by akash-rajak** [comment]
Hi,
        Thanks for reaching out, and I appreciate that you loved
the project.

         I have just used the tensorflow library, I thought and
implementing Yolo version 4 using GPU, but did not implemented

Thanks.
